Codeinwp/neve
# Allow custom colors to be selected inside customizer

### Summary
- Got the colors from CustomGlobalColors inside the CustomPalette component
- Changed the popup's max height to 410px
- Added the CSS vars inside the customizer, too, so we can display the color inside the popup

### Will affect the visual aspect of the product
NO

### Screenshots 
![Screen Capture on 2023-02-07 at 17-20-56 (1)](https://user-images.githubusercontent.com/9929553/217287436-91c87adc-14c5-45c1-9624-4b37fe460b88.gif)

### Test instructions
- Add a Custom global color and check if it appears with the other global colors. (Eg. Click on the globe next to the background color of a header row and see if the color is in that popup )
- Click on that color, save the customizer, and see if it works on the frontend
- See if the element color is changed when you change the custom global color you've added
- Delete the custom global color and see if it gets deleted wherever it's used.

- Please test that the colors inside the mega menu work as expected. Please use this PR when trying them: https://github.com/Codeinwp/neve-pro-addon/pull/2375


### Time
~ 11h 13min


Closes Codeinwp/neve-pro-addon#2341.
